tests/unit/os_test.ts

```diff
@@ -196,6 +197,22 @@ Deno.test({ permissions: { read: false } }, function execPathPerm() {
   );
 });
 
+Deno.test(
+  {
+    ignore: Deno.build.os !== "linux",
+    permissions: { read: true },
+  },
+  function procRequiresAllowAll() {
+    assertThrows(
+      () => {
+        Deno.readTextFileSync("/proc/net/dev");
+      },
+      Deno.errors.PermissionDenied,
+      `PermissionDenied: Requires all access to "/proc/net/dev", run again with the --allow-all flag`,
+    );
+  },
+);
```

Comment:
I tried it out and `cargo test js_unit_tests::os_test` runs fine with this, so I reverted to it.

Reply:
Hmmm... it was passing locally and now it's failing. I probably wasn't running it correctly. I'm looking into this.

Reply:
I'm going to revert my changes and follow up in a separate pr